src/sequelize.js

```diff
@@ -685,6 +645,73 @@ Use Sequelize#query if you wish to use replacements.`);
     }, retryOptions);
   }
 
+  /**
+   * Converts a connection string into an object with connection properties
+   *
+   * @param {string} connectionString string value to convert
+   * @param {object} options an optional parameter to use an existing options object
+   *
+   * @returns {object}
+   */
+  parseConnectionString(connectionString, options) {
```

Comment:
Would you mind moving this function to `src/utils/url.ts`? I'm trying to reduce the size of `sequelize.js` and we can consider this function to be internal :)